packages/ftp/Dockerfile

```diff
@@ -15,12 +15,18 @@ USER appuser
 
 # Copy the application code as the non-root user
 COPY --chown=appuser:appuser . .
+USER root
```

Comment:
Why we need user root here?

Reply:
I Think we can use the user `root` to mount the secret to /root/.npmrc, and then run `yarn install`. After that, I change to user appuser to build, so if someone hijack our image he can't see the secret file npmrc which is mount by root (also, when mounting as secret, the file is empty after the image building is done).

Reply:
lgtm